### PR TITLE
Add pytorch-ignite with test

### PR DESCRIPTION
**Pull request by vfdev-5**
Adds `ignite` recently released library to help with training neural networks in PyTorch
More info [here](https://pytorch.org/ignite/)

- docker image built and tested as described in README

### CODE REVIEW

**Comment by crawforc3** [comment]
Thanks! I'll be testing this out today.

**Comment by vfdev-5** [comment]
@crawforc3 I already tested it as it is described in the README:
- I ran the script `./build`
- I ran the script `./tests`

Everything goes correctly I would say, here is output of `./test`:

<details>
   <summary>click</summary>

```
docker-python$ ./test 
Error: No such container: jupyter_test
123222a493665d097ab3d3e10f32f62881f65c4fb6019bcfe7dab47e72d3df65
jupyter_test
jupyter_test
1.9.0-rc0
b'TensorFlow ok'
Tensorflow ok
Using TensorFlow backend.
Keras ok
tensor([[ 0.8517, -0.6919,  0.2332],
        [ 0.3050, -0.8079, -0.3354]])
PyTorch ok
fast.ai ok
ignite ok
Numpy imported ok
Your lucky number is: 27
Pandas imported ok
sklearn imported ok
sklearn RandomForestClassifier: ok
sklearn LinearRegression: ok
xgboost XGBClassifier: ok
matplotlib.pyplot ok
Basemap ok
plotly ok
Theano ok
nltk ok
OpenCV ok
skimage ok
wordbatch ok
pyfasttext ok
fastText ok
mxnet ok
bokeh ok
seaborn ok
127.0.0.1 - - [23/Jun/2018 13:56:23] "GET /bigquery/v2/projects/test/datasets HTTP/1.1" 200 -
bigquery proxy ok
shap ok
kmapper ok
```
</details>

**Comment by crawforc3** [comment]
Awesome, thanks. I still need to test it on our end before I push it into
production is all

**Comment by vfdev-5** [comment]
@crawforc3 any updates on this ?

**Comment by aprotopopov** [comment]
@vfdev-5 Hey, I'm also missing ignite in kaggle kernels... Why did you close that ticket? Is it solved somewhere else?
@crawforc3 are you going to add ignite to docker-python?

**Comment by vfdev-5** [comment]
@aprotopopov I have no idea why there were no actions on the PR since 06/2018...

**Comment by aprotopopov** [comment]
Yes, but in current [jigsaw challenge](https://www.kaggle.com/c/jigsaw-unintended-bias-in-toxicity-classification) internet is restricted. I added pytorch wheel 0.2.0 as [kaggle dataset](https://www.kaggle.com/protan/pytorch-ignite-020) for now. But I hope that ignite will be introduced later by default.

**Comment by aprotopopov** [comment]
@vfdev-5 Could you reopen issue? Or maybe we could write somewhere else about supporting pytorch-ignite..

**Comment by vfdev-5** [comment]
@aprotopopov

**Comment by vfdev-5** [comment]
@Philmod could you please review and give a feedback whether this PR can be merged ? Thanks

**Comment by Philmod** [comment]
@vfdev-5 Can you merge the changes from master into your branch? Otherwise, I tested your change and it looks good :-)

**Comment by vfdev-5** [comment]
@Philmod I merged the master, please check if it is OK for you.

File: Dockerfile [hunk 1]
```diff
@@ -490,6 +490,8 @@ RUN pip install flashtext && \
     pip install chainercv && \
     pip install plotly_express && \
     pip install albumentations && \
+    pip install allennlp && \
```
Comment (Philmod) [inline Dockerfile]: This line should be removed

Comment (vfdev-5) [reply Dockerfile]: Done

**Pull request merged**